IRC.md

```diff
@@ -335,6 +336,7 @@ Changes only contain the relevant tag. Setting slow mode to 10 seconds for examp
 - `r9k` is R9K mode. Messages with more than 9 characters must be unique. `0` means disabled, `1` enabled.
 - `subs-only` is subscribers only mode. Only subscribers and moderators can chat. `0` disabled, `1` enabled.
 - `slow` determines how many seconds chatters without moderator privileges must wait between sending messages.
+- `followers-only` is follower only mode. Only followers and moderators can chat. `-1` disabled, `0` enabled.
```

Comment:
This is not correct. If you set the duration argument to the /followers command, this is a positive number.

On a related note, I'm not sure pull requests on this repository are still considered, since it has been stressed that the official documentation on dev.twitch.tv is not in relation to this repository.